src/nodedef.cpp

```diff
@@ -339,15 +344,22 @@ void ContentFeatures::deSerialize(std::istream &is)
 		groups[name] = value;
 	}
 	drawtype = (enum NodeDrawType)readU8(is);
+
+	bool ignore_culling = ((version <= 26) &&
+			((drawtype == NDT_MESH) ||
```

Comment:
is this possible to replace this with (drawtype >= thing && drawtype <= thing) to remove two conditions ?